activity_browser/ui/tables/activity.py

```diff
@@ -102,7 +130,9 @@ def __init__(self, parent=None):
         self.setItemDelegateForColumn(0, FloatDelegate(self))
         self.setItemDelegateForColumn(1, StringDelegate(self))
         self.setItemDelegateForColumn(2, StringDelegate(self))
-        self.setItemDelegateForColumn(3, FormulaDelegate(self))
+        self.setItemDelegateForColumn(3, StringDelegate(self))
```

Comment:
Shouldn't this be a new `BooleanDelegate`? Or am I missing something somewhere else? Sorry if so...

Reply:
Good catch! I just saw this today. In fact as I used the built-in checkbox delegate, no delegate needs to be set here.

Reply:
Even better :)